app/views/admin/schools/bill_requests/new.html.erb

```diff
@@ -58,6 +58,9 @@
        <div class="col">
          <div class="mt-4">
            <%= f.button :submit, "Request bill", "data-confirm": 'Are you sure?', disabled: @users.empty? %>
+           <% if @school.bill_requested_at %>
+             <span class='p-2'>Bill requested <%= @school.bill_requested_at.strftime("on %m/%d/%Y at %H:%M") %></span>
```

Comment:
Could you tweak this a bit, to say "Bill last requested from the school on...at...